### ISSUE TITLE
spectaql 2.0.0 can't be installed with npm

### ISSUE THREAD

**Issue by cosban** [anvilco/spectaql#603]
I am unable to install this tool after 2.0.0 was released. Older versions install just fine.

```bash
cosban@quasar:/Users/cosban/dev $
> npm --version
8.19.2
cosban@quasar:/Users/cosban/dev $
> node --version
v18.12.1
cosban@quasar:/Users/cosban/dev $
> npm i -g spectaql@2.0.0
npm WARN deprecated opn@6.0.0: The package has been renamed to `open`
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm ERR! code 127
npm ERR! path /Users/cosban/.nvm/versions/node/v18.12.1/lib/node_modules/spectaql
npm ERR! command failed
npm ERR! command sh -c -- yarn build:vendor
npm ERR! sh: yarn: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cosban/.npm/_logs/2022-12-07T19_28_26_900Z-debug-0.log
```

**Comment by newhouse** [comment]
For now, can you please install yarn on your machine and try again?

**Comment by cosban** [comment]
Yes, doing so works and allows the install to occur.

**Comment by cosban** [comment]
```bash
cosban@quasar:/Users/cosban/dev $
> npm install -g yarn

added 1 package, and audited 2 packages in 464ms

found 0 vulnerabilities
cosban@quasar:/Users/cosban/dev $
> npm install -g spectaql@2.0.0
npm WARN deprecated opn@6.0.0: The package has been renamed to `open`
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.

added 13 packages, removed 62 packages, changed 332 packages, and audited 346 packages in 7s

37 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by newhouse** [comment]
Ok I'll get a fix out asap.

**Comment by newhouse** [comment]
@cosban this should be fixed in `main` and `spectaql@2.0.1`. Can you remove `yarn` and try again to confirm?

**Comment by cosban** [comment]
Thanks for the quick turn around. I have confirmed that 2.0.1 resolves this issue.
